target/deploy/uxd-keypair.json

```diff
@@ -1 +1,5 @@
-[82,32,63,76,54,228,185,166,87,235,177,224,36,208,169,222,36,78,156,110,191,97,17,216,129,160,100,242,206,72,36,26,31,23,37,22,77,200,201,4,42,8,149,53,79,63,23,45,62,201,158,193,151,104,170,21,145,73,30,135,7,113,204,230]
\ No newline at end of file
+[
```

Comment:
note before merge: revert this

Reply:
this keypair is updated to match with the devnet program id